addons/website/views/snippets/s_pricelist_boxed.xml

```diff
@@ -95,7 +95,7 @@
     </xpath>
     <xpath expr="." position="inside">
         <div data-js="Pricelist" data-selector=".s_pricelist_boxed">
-            <we-checkbox string="Descriptions" data-toggle-description="true" data-items-class="s_pricelist_boxed_item" data-description-class="s_pricelist_boxed_item_description" data-no-preview="true"/>
+            <we-checkbox string="Descriptions" data-toggle-description="true" data-items-class="s_pricelist_boxed_item" data-description-class="s_pricelist_boxed_item_description" data-description-extra-class="o_small" data-no-preview="true"/>
```

Comment:
Since we are in stable, existing users will not have this change, you can keep it, but you should find a JS way to add the class too :wink: 
Maybe just a `if is  ".s_pricelist_boxed, .s_pricelist_cafe"` in the option, with a brief explanation (since we are in website anyway) :slightly_smiling_face: